drivers/video/video_st_mipid02.c

```diff
@@ -189,7 +189,7 @@ static int mipid02_get_fmt(const struct device *dev, struct video_format *fmt)
 		}
 
 		desc = mipid02_get_format_desc(fmt->pixelformat);
-		if (desc) {
+		if (desc == NULL) {
```

Comment:
[non-blocking]
I notice the rest of the code uses `if (!desc)`, so if wanting to keep the change similar to the rest of the code:
```suggestion
		if (!desc) {
```

Reply:
Noted, but in my last PR a reviewer suggested using == NULL instead of !var for clarity and MISRA compliance. I used the same here to stay consistent. Let me know if you'd prefer it changed.

Reply:
Thanks for this patch. This is weird that I have been able to use this with this typo in the code ....
Anyway I'm all ok with this modification.

Overall the driver is (trying) to follow MISRA rule. There seems to have some NULL ptr check that are still done using !ptr but the trend is to have proper MISRA compliant code so it is probably better to use desc == NULL as you did in this patch.

Reply:
Agreed, thanks